internal/database/sqlite.go

```diff
@@ -113,6 +113,27 @@ PRAGMA foreign_keys=on;`,
 			"add_holder_name_to_accounts",
 			`alter table accounts add column holder_name default '';`,
 		),
+		execsql(
+			"create_validations",
+			`create table if not exists validations(
+				validation_id primary key,
+				account_id,
+				status,
+				strategy,
+				vendor,
+				created_at datetime,
+				updated_at datetime
+			);`,
+		),
+		execsql(
+			"create validations index",
+			`
+			create unique index
+				idx_validations_validation_account_ids
+			on
+				validations (validation_id, account_id);
```

Comment:
`validation_id` is already unique from being a `primary key` and wouldn't we want the uniqueness criteria to be `(strategy, vendor, account_id)`? 

That would allow us to enforce only one (instant, plaid) validation could exist for an Account.

Reply:
I created this index to sped up validation lookup (by validation id and account id). I don't think we need to limit validations (make strategy vendor unique for validation). If, for some reasons user fails to complete the validation (let's say widget/browser error) then he will be able to initiate a new validation. This may be the issue only for micro-deposits, but we handle this case inside micro-deposits strategy by not allowing to create micro-deposits if they were already created.